backend/middlewares/validators/testValidators.ts

```diff
@@ -113,6 +114,23 @@ export const questionsValidator = (
         if (!validatePrimitive(questionComponent.metadata.answer, "number")) {
           throw new Error("The answer field is not of type number");
         }
+      } else if (questionComponent.type === QuestionComponentType.FRACTION) {
+        if (!("numerator" in questionComponent.metadata)) {
+          throw new Error("Fraction component is missing a numerator field");
+        }
+        if (
+          !validatePrimitive(questionComponent.metadata.numerator, "number")
```

Comment:
```suggestion
          !validatePrimitive(questionComponent.metadata.numerator, "integer")
```
can you make this change for both fields? also for line 82 and 104